sandbox/pages/StatusCommunityTagsPage.qml

```diff
@@ -0,0 +1,87 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+
+import Sandbox 0.1
+
+Item {
+
+    ColumnLayout {
+        width: 600
+        anchors.centerIn: parent
+
+        Flow {
+            spacing: 10
+            Layout.fillWidth: true
+            
+            Repeater {
+                model: [
+                   "✊ Activism",
+                   "🎨 Art",
+                   "🔗 Blockchain",
+                   "📚 Books & blogs",
+                   "💼 Career",
+                   "🤝 Collaboration",
+                   "🛒 Commerce",
+                   "Ξ Crypto",
+                   "🎎 Culture",
+                   "🚀 DAO",
+                   "📈 DeFi",
+                   "🧩 Design",
+                   "🔨 DIY",
+                   "🌿 Environment",
+                   "🎒 Education",
+                   "🍿 Entertainment",
+                   "🗓 Event",
+                   "🧙‍♂️ Fantasy",
+                   "🧦 Fashion",
+                   "🌶 Food",
+                   "🎮 Gaming",
+                   "🌍 Global",
+                   "🧠 Health",
+                   "📐 Hobby",
+                   "🧪 Innovation",
+                   "📜 Language",
+                   "✨ Lifestyle",
+                   "📍 Local",
+                   "❤️ Love",
+                   "💎 Markets",
+                   "🎞 Movies & TV",
+                   "🎶 Music",
+                   "🗞 News",
+                   "🖼 NFT",
+                   "🙏 Non-profit",
+                   "🍆 NSFW",
+                   "🏢 Org",
+                   "🐶 Pets",
+                   "🎲 Play",
+                   "🎙️ Podcast",
+                   "🗳️ Politics",
+                   "🍱 Product",
+                   "🍁 Psyche",
+                   "👻 Privacy",
+                   "🔒 Security",
+                   "☕ Social",
+                   "👩‍💻 Software dev",
+                   "⚽️ Sports",
+                   "📱 Tech",
+                   "🗺 Travel",
+                   "🚕 Vehicles",
+                   "🌐 Web3"
+                ]
+
+                delegate: StatusFlatButton {
+                    size: StatusBaseButton.Size.Tiny
```

Comment:
I didn't know we had a tiny variant - nice!

If we want to have an actual `StatusCommunityTag` component we can introduce it using this definition of `StatusFlatButton` here

Reply:
Ok, i'll do it